python/empty/README.md

```diff
@@ -1,6 +1,6 @@
-# Empty template
+# Sample template
 
-[This project]() is an empty template for Python applications using the Quix serverless environment.
+[This project](https://github.com/quixio/quix-samples/tree/main/python/empty) is a template for Python based Quix Samples using the Quix serverless environment.
```

Comment:
> is a template for Python based Quix Samples

Should be Python-based. Perhaps better as:

...is a Python template for the Quix serverless environment.

Note: probably other instances too.